### DISCUSSION TITLE
_carousel.scss /* rtl:begin:ignore */

### DISCUSSION THREAD

**Discussion by chovanjo** [twbs/bootstrap#36552] [General]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Hi Team, you use in _carousel.scss /* rtl:begin:ignore */, but when I use RTLCSS on Bootstrap.scss it fails on "rtlcss: found end "ignore" without a matching begin." It looks it's because RTLCSS doesn't like inner directives.

```
/* rtl:begin:ignore */
    /* rtl:begin:ignore */
    /* rtl:end:ignore */
/* rtl:end:ignore */
```

Any idea how to resolve that.

### Reduced test cases

Repo: https://github.com/zeroamps/examples/tree/main/mean/13_ltr-rtl.

See: https://github.com/zeroamps/examples/blob/main/mean/13_ltr-rtl/src/styles.scss
See: https://github.com/zeroamps/examples/blob/main/mean/13_ltr-rtl/src/themes/hello.scss

Commented // @import '~bootstrap/scss/carousel'; it works.
Uncommented it doesn't work.

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

v5.1.3

**Comment by mdo** [comment]
I think you're better served via a Discussion here or via Slack. This doesn't sound like a bug in Bootstrap itself, but rather your implementation.
